upi/vsphere/README.md

```diff
@@ -1,59 +1,47 @@
-# Pre-Requisites
+# Installing OpenShift on VMware
 
-* terraform
-* jq
+The terraform examples provided require a AWS account.  AWS is used
+for its API interface to network load balancing and DNS.
+The `./modules/rhcos_virtual_machines` can be used independently if so desired.
 
-# Build a Cluster
+We will begin with using `openshift-installer` as each terraform example 
+starts with creating the ignition files.
 
-1. Create an install-config.yaml.
-The machine CIDR for the dev cluster is 139.178.89.192/26.
+## Create ignition files for OpenShift RHCOS virtual machines
```

Comment:
nit: We can probably drop "for OpenShift RHCOS virtual machines" while remaining sufficiently specific.  For comparison, AWS and GCP docs have "Create Ignition configs" (they disagree on Configs vs. configs, I don't think we have a clear title vs. sentence case policy for h2+ ;).